articles/Setup-Windows-Python.md

```diff
@@ -15,7 +15,7 @@ ms.devlang:   python
 
 Installing CNTK for Python on your system requires the installation of some third party packages and the definition of the PATH  environment variable.
 
-### GPU Specific Packages
+### GPU-Specific Packages
```

Comment:
We could prepend "OPTIONAL. " to the title just like we did for OpenCV because user requires CUDA only if it intends to have GPU support